oarepo_ui/templates/oarepo_ui/footer.html

```diff
@@ -31,7 +31,7 @@
           {%- block footer_bottom %}
           <div class="eight wide column left middle aligned">
             {%- block footer_bottom_left %} {% trans
-            link="http://inveniosoftware.org" %}{{_("Powered by")}}
+            link="http://inveniosoftware.org" %}Powered by
             <a href="{{ link }}">Invenio</a>{% endtrans %} {%- endblock
```

Comment:
We would like to have this translated

Reply:
@mirekys It is already inside of this trans block, which should allow for the translation of this section. This gets extracted to the PO file, but to be honest, when I tried providing it the translation string, it did not quite work. I am not sure if there is some catch.

Reply:
You are right I missed the trans block completely 🙈

Reply:
Tried to fix it but also no success. But don't think this trans block is a blocking issue :)